qplaintexteditsearchwidget.cpp

```diff
@@ -137,6 +138,13 @@ bool QPlainTextEditSearchWidget::eventFilter(QObject *obj, QEvent *event) {
 
 void QPlainTextEditSearchWidget::searchLineEditTextChanged(
     const QString &arg1) {
+
+    QSettings settings;
+    if (arg1.size() != 0 &&
+        settings.value(QStringLiteral("searchMinCharacters"), 0).toInt() > arg1.size()) {
```

Comment:
We can't depend on some option from QOwnNotes. This library is independent of QOwnNotes. As such, there is no need for this code here.